modules/virt-about-qemu-guest-agent-web.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-viewing-qemu-guest-agent-information.adoc
+
+[id="About-the-qemu-guest-agent-web_{context}"]
+= About the QEMU guest agent information in the web console
+
+When the QEMU guest agent is installed, the *Details* pane within the *Virtual Machine Overview* tab and the *Details* tab display information about the *Hostname*, *Operating system*, *Time zone*, and *Logged in users*.
+
+The *Virtual Machine Overview* shows information about the guest operating system installed on the virtual machine.
+
+The *Details* tab displays a table for *Logged in users.* This information includes *User name*, *Domain*, *Login time*, and *Elapsed Logged in time*.
+
+The *Disks* tab displays a table for *File systems*. This information includes *Name*, *File system type*, *Mount point*, *Total bytes*, and *Used bytes*.
```

Comment:
Same comment about the GUI items.

```
 The *Disks* tab displays a table for file systems. This information includes the name, file system type, mount point, total bytes, and used bytes.
```

And whether it's really necessary to call out the fields.

Reply:
Resolved by removing the bolded items, and mentioning only the table for file system.s